mask_flags.py

```diff
@@ -90,16 +102,18 @@ def valid_argv(argv):
 
 def run(cmd):
     '''Prepare and run a subprocess cmd, returning a CompletedProcess.'''
+    print("Preparing the following cmd:\n", " ".join(cmd))
     cmd = prepare_subprocess_cmd(cmd)
-    print("Running the following cmd", cmd)
-    return subprocess.run(cmd, stdout=sys.stdout, stderr=subprocess.PIPE)
+    print("Running the following cmd:\n", " ".join(cmd))
```

Comment:
Should probably leave this as a list, not as a joined string, since arguments with spaces might print misleadingly even though subprocess was correctly escaping them.

Reply:
with a cmd that wraps three lines, it's really not easier at all...